modules/gltf_blend/editor_scene_importer_blend.cpp

```diff
@@ -0,0 +1,116 @@
+#if TOOLS_ENABLED
+#include "editor_scene_importer_blend.h"
+
+#include "core/core_bind.h"
+#include "core/object/ref_counted.h"
+#include "modules/gltf/editor_scene_importer_gltf.h"
+#include "scene/3d/node_3d.h"
+#include "scene/animation/animation_player.h"
+#include "scene/resources/animation.h"
+
+uint32_t EditorSceneFormatImporterBlend::get_import_flags() const {
+	return ImportFlags::IMPORT_SCENE;
+}
+
+void EditorSceneFormatImporterBlend::get_extensions(List<String> *r_extensions) const {
+	r_extensions->push_back("blend");
+}
+
+Node *EditorSceneFormatImporterBlend::import_scene(const String &p_path,
+		uint32_t p_flags, int p_bake_fps,
+		List<String> *r_missing_deps,
+		Error *r_err) {
+	String addon_path = EditorSettings::get_singleton()->get_setting("filesystem/blend/blender_path");
+	String scene_to_blend_setting = "import bpy, os, sys;"
+									"bpy.ops.wm.open_mainfile(filepath='GODOT_SOURCE');"
+									"(bpy.ops.export_scene.gltf("
+									"filepath='GODOT_SINK',"
+									"export_format='GLTF_SEPARATE',"
+									"export_keep_originals=True,"
+									"export_texture_dir='GODOT_TEXTURE_DIR',"
+									"export_colors=True,"
+									"export_all_influences=False,"
+									"export_extras=True,"
+									"export_cameras=True,"
+									"export_lights=True)) if bpy.app.version >= (3, 0, 0) else "
+									"(bpy.ops.export_scene.gltf("
+									"filepath='GODOT_SINK',"
+									"export_format='GLTF_SEPARATE',"
+									"export_texture_dir='GODOT_TEXTURE_DIR',"
+									"export_colors=True,"
+									"export_all_influences=False,"
+									"export_extras=True,"
+									"export_cameras=True,"
+									"export_lights=True));";
+	String script = scene_to_blend_setting;
+	String source = ProjectSettings::get_singleton()->globalize_path(p_path);
+	String source_global = ProjectSettings::get_singleton()->globalize_path(source);
+	source_global = source_global.c_escape();
+	String texture_global = "res://.godot/imported/" + p_path.get_file().get_basename() + "-" + p_path.md5_text() + "_textures";
+	texture_global = ProjectSettings::get_singleton()->globalize_path(texture_global);
+	texture_global = texture_global.c_escape();
+	script = script.replace("GODOT_TEXTURE_DIR", texture_global);
+	script = script.replace("GODOT_SOURCE", source_global);
+	String sink = "res://.godot/imported/" + p_path.get_file().get_basename() + "-" + p_path.md5_text() + ".gltf";
+	String sink_global = ProjectSettings::get_singleton()->globalize_path(sink);
+	sink_global = sink_global.c_escape();
+	script = script.replace("GODOT_SINK", sink_global);
+	Vector<String> args;
+	args.push_back("--background");
+	args.push_back("--python-expr");
+	args.push_back(script);
+	print_line(script);
+	Array standard_out;
+	int32_t ret = core_bind::OS::get_singleton()->execute(addon_path, args, standard_out, true);
+	for (int32_t line_i = 0; line_i < standard_out.size(); line_i++) {
+		print_line(standard_out[line_i]);
```

Comment:
Do we still want to leave the (non-error)`print`s in? It might be a bit noisy compared to other importers. Here's a sample:

```
Godot Engine v3.4.stable.arch_linux (c) 2007-2021 Juan Linietsky, Ariel Manzur & Godot Contributors.
--- GDScript language server started ---
[--background, --python-expr, import bpy, os, sys;bpy.ops.wm.open_mainfile(filepath='/home/rcorre/src/godot-blender/example.blend');bpy.ops.export_scene.gltf(filepath='/home/rcorre/src/godot-blender/.import/example.blend-1ba6972ae2a764efa4ba948c0919249c.glb',export_format='GLB',export_colors=True,export_all_influences=False,export_extras=True,export_cameras=True,export_lights=True);]
'/usr/bin/2.93/python/lib/python3.9/site-packages/libextern_draco.so' does not exist, draco mesh compression not available
11:59:01 | INFO: Starting glTF 2.0 export
11:59:01 | INFO: Extracting primitive: Cube
11:59:01 | INFO: Primitives created: 1
11:59:01 | INFO: Finished glTF 2.0 export in 0.0035049915313720703 s

Blender 2.93.5 (hash a791bdabd0b2 built 2021-11-06 11:01:22)
Read prefs: /home/rcorre/.config/blender/2.93/config/userpref.blend
/run/user/1000/gvfs/ non-existent directory
Read blend: /home/rcorre/src/godot-blender/example.blend

Blender quit
```

Reply:
Debugging blender was difficult without this.

Reply:
Well once it's working, this would still be too verbose. If it's only for debugging, it should be `print_verbose` at the very least.